README.md

```diff
@@ -21,28 +34,20 @@ This project can be used to quickly setup Satellite 6.1 or 6.2 install with the
     ```
    Note: The ansible playbook run will fail and not proceed if the free space in root partition is less than the value specified in `required_root_free_space` variable in `roles/sat6repro/vars/main.yml`
 
-### Workstation requirements:
-3. Install `ansible` package in your workstation. For RHEL boxes, [access to EPEL] (https://access.redhat.com/solutions/3358) is required.
+#### On Workstation
 
-    ```console
-     # yum install -y ansible
-    ```
-4. git clone this project in your workstation.
-
-    ```console
-     # git clone git@github.com:RedHatSatellite/satellite-clone.git
-    ```
-5. Download the data backup tar files - config, pgsql, mongodb in your workstation under the project folder - satellite-clone/roles/sat6repro/files so Ansible can find them.
-6. Update the required sections in `roles/sat6repro/vars/main.yml`.
-7. Update the `inventory` file under the root of this project to add your Satellite VM's ip address.
-8. Run the ansible playbook:
+1. Download the data backup tar files - config, pgsql, mongodb in your workstation under the project folder - satellite-clone/roles/sat6repro/files so Ansible can find them.
+2. Update the required sections in `roles/sat6repro/vars/main.yml`.
+3. Update the `inventory` file under the root of this project to add your Satellite VM's ip address. If executing the playbook on localhost, add `ansible_connection=local` after the IP address.
```

Comment:
Can we say "Add the IP address of the blank machine you are cloning to"? The machine is not a Satellite machine yet.